MapShare/MapShare/UI/Active Session View/ActiveSessionViewController.swift

```diff
@@ -0,0 +1,65 @@
+//
+//  ActiveSessionViewController.swift
+//  MapShare
+//
+//  Created by iMac Pro on 5/3/23.
+//
+
+import UIKit
+
+class ActiveSessionViewController: UIViewController {
+
+    //MARK: - OUTLETS
+    @IBOutlet weak var sessionNameLabel: UILabel!
+    @IBOutlet weak var sessionCodeLabel: UILabel!
+    @IBOutlet weak var sessionControlButton: UIButton!
+    @IBOutlet weak var activeSessionTableView: UITableView!
+    
+    
+    //MARK: - PROPERTIES
+    var activeSessionViewModel: ActiveSessionViewModel!
+    
+    
+    //MARK: - LIFECYCLE
+    override func viewDidLoad() {
+        super.viewDidLoad()
+//        activeSessionViewModel = ActiveSessionViewModel()
```

Comment:
Is this intentionally left in? Unused code should be removed

Reply:
Yes, it was a note to myself and my partner that the ViewModel would need to be initialized, however I knew that I would handle that via segue Navigation.

In future cases, I'll use my #warning("")